api-module-library/microsoft-teams/manager.js

```diff
@@ -0,0 +1,185 @@
+const { debug, flushDebugLog } = require('@friggframework/logs');
+const { get } = require('@friggframework/assertions');
+const { ModuleManager } = require('@friggframework/module-plugin');
+const { Api } = require('./api/api');
+const { graphApi } = require('./api/graph');
+const { botFrameworkApi } = require('./api/botFramework');
+const { Entity } = require('./models/entity');
+const { Credential } = require('./models/credential');
+const config = require('./defaultConfig.json');
+
+class Manager extends ModuleManager {
+    static Entity = Entity;
+    static Credential = Credential;
+
+    constructor(params) {
+        super(params);
+    }
+
+    //------------------------------------------------------------
+    // Required methods
+    static getName() {
+        return config.name;
+    }
+
+    static async getInstance(params) {
+        const instance = new this(params);
+        // All async code here
+
+        // initializes the Api
+        let teamsParams = {
+            client_id: process.env.TEAMS_CLIENT_ID,
+            client_secret: process.env.TEAMS_CLIENT_SECRET,
+            redirect_uri: process.env.TEAMS_REDIRECT_URI,
```

Comment:
This one we've been using the paradigm of one common REDIRECT_URI and just appending `/microsoft-teams` to it... breaks for some folks, but the idea lets the adopter be consistent and set up one oauth redirect handler.

So `${process.env.REDIRECT_URI}/microsoft-teams`